var/spack/repos/builtin/packages/dakota/package.py

```diff
@@ -60,4 +68,7 @@ def cmake_args(self):
                 '-DMPI_CXX_COMPILER:STRING=%s' % join_path(spec['mpi'].mpicxx),
             ])
 
-        return args
+        if '+hdf5' in spec:
+            args.extend(['DAKOTA_HAVE_HDF5:BOOL=ON'])
```

Comment:
```suggestion
            args.append('DAKOTA_HAVE_HDF5:BOOL=ON')
```
Can you explicitly disable this in an else clause?

Reply:
I explicitly disabled this in an else clause with the same syntax as shared libraries.